apps/io-session-manager/src/app.ts

```diff
@@ -160,16 +157,14 @@ export const newApp: (
 
   const acsDependencies: AcsDependencies = {
     redisClientSelector: REDIS_CLIENT_SELECTOR,
-    isLollipopEnabled: FF_LOLLIPOP_ENABLED,
     appInsightsTelemetryClient: appInsightsClient,
     getClientErrorRedirectionUrl,
     getClientProfileRedirectionUrl,
     isUserElegibleForIoLoginUrlScheme,
-    FF_UNIQUE_EMAIL_ENFORCEMENT_ENABLED,
     isSpidEmailPersistenceEnabled:
       LoginConfig.IS_SPID_EMAIL_PERSISTENCE_ENABLED,
     testLoginFiscalCodes: LoginConfig.TEST_LOGIN_FISCAL_CODES,
-    hasUserAgeLimitEnabled: LoginConfig.FF_USER_AGE_LIMIT_ENABLED,
+    hasUserAgeLimitEnabled: true,
```

Comment:
We could completely remove it, like we did for the other FFs.

Reply:
removed in https://github.com/pagopa/io-auth-n-identity-domain/pull/99/commits/6b6770a540af58eeb03e00ad1d751eb8583d9262